projects/observability/src/shared/graphql/request/handlers/entities/query/entities-graphql-query-builder.service.ts

```diff
@@ -92,44 +88,20 @@ export class EntitiesGraphqlQueryBuilderService {
   }
 
   public normalizeEntity(rawResult: Dictionary<unknown>, request: EntityDescriptor): Observable<Entity> {
-    return forkJoinSafeEmpty(
```

Comment:
This logic was repeated in entities handler, interaction handler and is supposed to be used in the new actor and actors handler. So I decided to move it to meta data service so that the code can be shared. 
@aaron-steinfeld what do you think? Will add some test after initial feedback.